components/Banner.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <header
+    class="w-full py-4 px-6 border-b border-ink-200 transition-background duration-500"
```

Comment:
The padding values should be half of this on mobile.

Reply:
These paddings are only effective in mobile view, and is in alignment with the content of the page below. If I reduce it to half, it doesn't look good.
<img width="444" alt="Screenshot 2021-02-12 at 5 56 19 PM" src="https://user-images.githubusercontent.com/68372876/107767997-cafd9500-6d5b-11eb-8055-e4551971317e.png">